packages/openfeature-web-provider/src/ConfidenceWebProvider.e2e.test.ts

```diff
@@ -1,106 +1,114 @@
-import { OpenFeature, ProviderEvents } from '@openfeature/web-sdk';
-import { createConfidenceWebProvider } from './factory';
-
-describe('ConfidenceHTTPProvider E2E tests', () => {
-  beforeAll(() => {
-    const confidenceProvider = createConfidenceWebProvider({
-      clientSecret: 'RxDVTrXvc6op1XxiQ4OaR31dKbJ39aYV',
-      timeout: 1000,
+import { ClientProviderStatus, OpenFeature, Provider, ProviderStatus } from '@openfeature/web-sdk';
+import { ConfidenceWebProviderOptions, createConfidenceWebProvider } from './factory';
+
+OpenFeature.setLogger({
+  error: jest.fn(),
+  warn: jest.fn(),
+  info: jest.fn(),
+  debug: jest.fn(),
+});
+function createProvider(options: Partial<ConfidenceWebProviderOptions> = {}): Provider {
+  return createConfidenceWebProvider({
+    clientSecret: 'RxDVTrXvc6op1XxiQ4OaR31dKbJ39aYV',
+    timeout: 1000,
+    ...options,
+  });
+}
+describe('ConfidenceWebProvider E2E tests', () => {
+  describe('initialize fail', () => {
+    beforeEach(async () => {
+      await expect(OpenFeature.setProviderAndWait(createProvider({ timeout: 0 }))).rejects.toThrow(
+        'This operation was aborted',
```

Comment:
This is the timeout error, should probably be something more meaningful. I.e. this hasn't been changed in this PR, just become visible through this test.